<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cleaner_Dialog</class>
 <widget class="QDialog" name="Cleaner_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	border-radius:10px;
	border-width:2px;
	
	background-color: rgb(107, 236, 255);
}

QPushButton:hover{
	
	background-color: rgb(105, 255, 88);
}

QLabel{
	border-width:2px;
	border-radius:10px;
	background-color: rgb(194, 255, 235);
	font: 14pt &quot;华文行楷&quot;;
}

QLabel:hover{
	
	background-color: rgb(255, 184, 234);
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>18</y>
     <width>801</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>请选择需要清理的盘符</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>请选择需要清理的垃圾</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="table_driver"/>
      </item>
      <item>
       <widget class="QTableWidget" name="table_rubbish"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="button_clean">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>20</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>清理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_fileDelete"/>
    </item>
    <item>
     <widget class="QProgressBar" name="clean_progress">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
